app/controllers/setup/two.js

```diff
@@ -169,11 +169,15 @@ export default Controller.extend(ValidationEngine, {
     _oauthSetup() {
         let blogTitle = this.get('blogTitle');
         let config = this.get('config');
+        let promises = [];
+
+        promises.pushObject(this.get('settings').fetch());
+        promises.pushObject(this.get('config').fetchPrivate());
 
         this.get('hasValidated').addObjects(['blogTitle', 'session']);
 
         return this.validate().then(() => {
-            return this.get('settings').fetch()
+            return RSVP.all(promises)
```

Comment:
It's probably also why the tests are failing, it's expecting a settings object in the `.then()` but `RSVP.all()` will return an array.